interface.idl

```diff
@@ -56,4 +56,8 @@ namespace net {
     oneway tx(payload: bytes);
 }
 
+namespace time {
```

Comment:
`time` is a bit confusing with `datetime` server which we'll work on later. How about `namespace rtc`?